src/DataTable/utils/searchText.js

```diff
@@ -0,0 +1,10 @@
+export const searchText = (text, rows) => {
+  if (text) {
+    return rows.filter((row) => {
+      return row.name1.toLocaleLowerCase().search(text.toLocaleLowerCase()) > -1 ||
```

Comment:
In general, 'search' function says exactly what is going on in here. However, different tests/different benchmarks sometimes shows that `indexOf` still might be faster. So depending what would be "real life" usage of `search` function and if every milisecond of the speed will be critical for the project, then it might be usefull to have our own tests on different browsers to verify which of the two is faster. Also, `search` do the converion to RegExp which sometimes may look like a little overkill for simple strings.

Reply:
It doesn't matter which is faster fo small data sets, and for larger datasets you won't use frontend anyway

Reply:
BTW I would extract comparison of two string to common function, because that's what is adding complexity and maintenance problem, not the filter itself.